### PR TITLE
Add support for international TAR forms

### PR DESCRIPTION
**Pull request by kberzinch**

### CODE REVIEW

File: app/Nova/Travel.php [hunk 1]
```diff
@@ -228,6 +228,87 @@ public function fields(Request $request): array
...
+                        ->rules('required_if:is_international,1', 'nullable')
+                        ->help(
+                            'Please explain how this travel meets essential travel criteria as defined by Georgia Tech.'
+                        )
+                        ->hideFromIndex(),
+
+                    Boolean::make('Export-Controlled Technology', 'export_controlled_technology')
+                        ->required()
+                        ->help(
+                            'Do you plan to take any information or technology that is controlled?'
```
Comment (RossIV) [inline app/Nova/Travel.php:251]: Is the assumption that the person completing this knows whether or not something they are planning to take is subject to export controls?

File: app/Nova/Travel.php [hunk 2]
```diff
@@ -228,6 +228,87 @@ public function fields(Request $request): array
...
+                        ->rules('required_if:export_controlled_technology,1', 'nullable')
+                        ->help(
+                            'If yes, please describe the information or technology.'
+                        )
+                        ->hideFromIndex(),
+
+                    Boolean::make('Embargoed Destination', 'embargoed_destination')
+                        ->required()
+                        ->help(
+                            'Do you plan to travel to an embargoed destination?'
```
Comment (RossIV) [inline app/Nova/Travel.php:266]: Is the assumption that the person completing this knows whether or not where they're going is an embargoed destination? Is that even allowed?

File: app/Nova/Travel.php (hunk 1 above)
Comment (kberzinch) [reply app/Nova/Travel.php:251]: Short answer yes. I don't think Nova help text is the place to explain all of this.

File: app/Nova/Travel.php (hunk 2 above)
Comment (kberzinch) [reply app/Nova/Travel.php:266]: Same as above. Not sure what you mean by the latter question.

Comment (RossIV) [reply app/Nova/Travel.php:266]: Are we/GT allowed to travel to embargoed destinations?

Comment (kberzinch) [reply app/Nova/Travel.php:266]: If you get a license from OFAC, sure.

**Review by RossIV — APPROVED** [review]

**Review by JoshOldenburg — APPROVED** [review]

File: app/Models/TravelAssignment.php [hunk 3]
```diff
@@ -225,7 +226,11 @@ public function getTravelAuthorityRequestUrlAttribute(): string
                 'non_contract_airline' => false,
             ])['non_contract_airline']
         ) {
-            return $this->getAirfarePowerFormUrl();
+            if ($this->travel->is_international) {
+                return $this->getInternationalAirfarePowerFormUrl();
+            } else {
+                return $this->getDomesticAirfarePowerFormUrl();
+            }
```
Comment (evan10s) [inline app/Models/TravelAssignment.php:233]: you could early exit here and drop the `else` braces

File: database/migrations/2023_02_11_172857_add_international_columns_to_travel.php [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('travel', static function (Blueprint $table): void {
+            $table->boolean('is_international');
+            $table->boolean('export_controlled_technology');
```
Comment (evan10s) [inline database/migrations/2023_02_11_172857_add_international_columns_to_travel.php:18]: Should the booleans for things that (seem like they) are only relevant to international travel also be nullable? Seems odd to set them to a value on a domestic trip, for instance

**Review by evan10s — APPROVED** [review]

**Review by JoshOldenburg — APPROVED** [review]

**Pull request merged**